docs/piwiks-extensibility-points.md

```diff
@@ -5,189 +5,8 @@ category: CoreDevelop
 
 Plugins can extend Piwik using the following methods:
 
-- using **events**
-- implementing **special classes** that are recognized by Piwik
-- extending certain **abstract base classes**
-
-This guide provides an overview describing how plugins can use these methods and some implementation details regarding how Piwik provides these methods.
-
-## Events
```

Comment:
I extracted this into a separate guide, and linked to it whenever we need to use events. It is also available within the develop menu